### ISSUE TITLE
Minor possible issue

### ISSUE THREAD

**Issue by Baza-1969** [jrnelson90/tinderboxpedal#2]
Just a quick one Justin - This is great work!.
Looks like your electrical schematic has the buttons going sequentially from bottom to top 1-4?
Button 1 and 2 look ok, but buttons 3 & 4 seem to be going to the wrong pins according to you GPIO assigment on Tinderbox.py (assuming the schematic buttons are drawn from 1 to 4 sequentially upwards)
Button 3 goes to 19 / button 4 goes to 26
This could cause confusion, with button 3 operating preset 4 and visa versa  - easily fixed with labelling the schematic buttons?
[](url)

**Comment by jrnelson90** [comment]
I agree, I did that schematic very quickly last night so I could go to bed. I'll be working on a much better one. I just need to reorient myself with Kicad or Eagle, it's been a while.
